Source/JavaScriptCore/dfg/DFGSpeculativeJIT64.cpp

```diff
@@ -2742,9 +2745,11 @@ void SpeculativeJIT::compileGetByVal(Node* node, const ScopedLambda<std::tuple<J
         if (node->arrayMode().isInBounds()) {
             SpeculateStrictInt32Operand property(this, m_graph.varArgChild(node, 1));
             StorageOperand storage(this, m_graph.varArgChild(node, 2));
+            GPRTemporary propertyTemp(this);
```

Comment:
We should not add new register allocation for GetByVal etc. as it regress code quality for the one of the hottest code. These things must be done only for ArrayAt.